### PR TITLE
COM-6264: Specify how to get and set up fluster on a local machine

### PR DESCRIPTION
**Pull request by german-martinez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,16 @@ In order to run the tests for the different test suites and decoders, a
 `resources` directory containing all the input files for each test suite needs
 to be collected first.
 
+## How to get started
+
+- Clone the repository: gh repo clone fluendo/fluster
```
Comment (rgonzalezfluendo) [inline README.md:66]:
```suggestion
- Clone the repository: `git clone https://github.com/fluendo/fluster.git` 
```

Better to use `git` than `gh`. It's more generic and nor only related with GitHub.

File: README.md [hunk 2]
```diff
@@ -61,6 +61,16 @@ In order to run the tests for the different test suites and decoders, a
 `resources` directory containing all the input files for each test suite needs
 to be collected first.
 
+## How to get started
+
+- Clone the repository: `git clone https://github.com/fluendo/fluster.git`
+
+or
+
+- [Release source code](https://github.com/fluendo/fluster/releases) assets is available in the following formats zip, tar.gz
```
Comment (mdimopoulos) [inline README.md:70]:
```suggestion
- [Release source code](https://github.com/fluendo/fluster/releases) assets are available in the following formats zip, tar.gz
```

**Review by mdimopoulos — CHANGES_REQUESTED** [review]
Minor grammatical issue

File: README.md [hunk 3]
```diff
@@ -16,6 +16,7 @@ suites.
 - [Fluster](#fluster)
   - [Table of Contents](#table-of-contents)
   - [Description](#description)
+  - [How to get started] (#how-to-get-started)
```
Comment (mdimopoulos) [inline README.md:19]:
```suggestion
  - [How to get started](#how-to-get-started)
```

**Review by mdimopoulos — CHANGES_REQUESTED** [review]
Minor change is needed to make sure README appears correctly

**Pull request merged**